cpp/tests/utilities/column_utilities.cu

```diff
@@ -46,14 +47,58 @@ void expect_column_properties_equal(cudf::column_view const& lhs, cudf::column_v
 }
 
 class corresponding_rows_unequal {
+  table_device_view d_lhs; 
+  table_device_view d_rhs;
 public:
-  corresponding_rows_unequal(table_device_view d_lhs, table_device_view d_rhs): comp(d_lhs, d_rhs) {
+  corresponding_rows_unequal(table_device_view d_lhs, table_device_view d_rhs)
+  : d_lhs(d_lhs),
+    d_rhs(d_rhs),
+    comp(d_lhs, d_rhs)
+  {}
+  
+  struct typed_element_unequal {
```

Comment:
Since everywhere it's being used is already checking equivalency, should I just rename this function?

Reply:
> Since everywhere it's being used is already checking equivalency

Not quite, this issue already came up before. Though I can't remember who... the issue was about the equivalence of a non-nullable column and a nullable column where all elements are valid.